.pre-commit-config.yaml

```diff
@@ -9,17 +9,41 @@ repos:
     rev: v1.0.0
     hooks:
       - id: go-fmt-goimports
-      - id: go-test
-      - id: go-vet
       - id: gofumpt
       - id: golint
       - id: staticcheck
-      - id: golangci-lint
+        files: ^lib/.*\.go$
   - repo: local
     hooks:
-      - id: prettier
-        name: Prettier
-        entry: bunx prettier --write --ignore-unknown
+      - id: go-test-lib
+        name: go-test (lib)
+        entry: bash -c 'cd lib && go test ./...'
```

Comment:
It's somewhat inconvenient that this now depends on *nix. How about `go test lib/...`? Same applies to the rest.